### PR TITLE
Updates per LB

### PR DESCRIPTION
**Pull request by cwisdo**

### CODE REVIEW

File: app/models/state_registrants/wa_registrant.rb [hunk 1]
```diff
@@ -381,17 +381,21 @@ def submit_to_online_reg_url
...
+      puts result.to_json
       self.wa_submission_error ||= []
       self.wa_submission_error << "#{DateTime.now}: No WA return_token"
       self.registrant.skip_state_flow!
       AdminMailer.wa_registration_error(self, self.wa_submission_error, "Registrant Switched to paper").deliver_now
     end
 
   rescue Exception=>e
+    puts "Unknown Error" 
+    puts result.to_json
```
Comment (amekelburg) [inline app/models/state_registrants/wa_registrant.rb:398]: Remove the puts lines?

File: app/models/state_customizations/wa.rb [hunk 2]
```diff
@@ -62,4 +63,8 @@ def self.permitted_attributes
     ].flatten
   end
 
+  def state_id_tooltip 
+    return "Foobaz" 
+  end
```
Comment (amekelburg) [inline app/models/state_customizations/wa.rb:68]: Is this needed?

**Review by amekelburg — CHANGES_REQUESTED** [review]

File: app/models/state_registrants/wa_registrant.rb (hunk 1 above)
Comment (cwisdo) [reply app/models/state_registrants/wa_registrant.rb:398]: Will do

Comment (cwisdo) [reply app/models/state_registrants/wa_registrant.rb:398]: have done

File: app/models/state_customizations/wa.rb (hunk 2 above)
Comment (cwisdo) [reply app/models/state_customizations/wa.rb]: Removed

**Pull request merged**